spec/cypress/integration/accessibility.js

```diff
@@ -76,4 +76,20 @@ describe("Accessibility", () => {
 
     cy.checkA11y();
   });
-});
+  // this test should only be ran locally due to the length of time taken to complete.
+  // to include it add '--env tags=checkCourseLessonsAccessibility'
+  // to the yarn cypress:open cmd
```

Comment:
I recommend capitalising comments (to make the reader feel like you've put enough work into them that they are worth reading, human psychology is a truly horrible thing). 

Also, add a space before comment, so it's obvious it refers to the test below.